modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/servlet/taglib/JSLoaderTopHeadDynamicInclude.java

```diff
@@ -60,49 +59,67 @@ public void include(
 			HttpServletResponse httpServletResponse, String key)
 		throws IOException {
 
-		PrintWriter printWriter = httpServletResponse.getWriter();
+		StringWriter stringWriter = new StringWriter();
```

Comment:
Why not use `printWriter` directly and avoid the unnecessary `_writeResponse` indirection? This was just added to account for the different branches between cached and fresh content. Should no longer be necessary.

Reply:
Yes :+1:

Reply:
done